### PR TITLE
MOTECH-2090 Remove server-api bundle

### PR DESCRIPTION
**Pull request by martokarski**
These classes were mainly used by motech-admin or motech-server bundle, so I move its to those packages.

### CODE REVIEW

File: platform/server-bundle/src/test/java/org/motechproject/server/web/controller/ModuleControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+/**
+ * Created by marek on 16.02.16.
```
Comment (sebbrudzinski) [inline platform/server-bundle/src/test/java/org/motechproject/server/web/controller/ModuleControllerTest.java]: When you get a chance to fix the tests, please also remove this javadoc.

File: platform/server-bundle/src/main/java/org/motechproject/server/ui/BundleService.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package org.motechproject.server.ui;
+
+import org.motechproject.server.web.dto.BundleIcon;
+
+/**
+ * The <code>BundleService</code> interface provides methods for getting icons from the bundles.
+ */
+public interface BundleService {
```
Comment (pgesek) [inline platform/server-bundle/src/main/java/org/motechproject/server/ui/BundleService.java]: Shouldn't this be called BundleIconService?

**Comment by martokarski** [comment]
Must be checked with modules

**Comment by martokarski** [comment]
None of modules use this bundle

**Comment by pgesek** [comment]
Needs rebase

**Pull request merged**